input/pagecontent/terminology.md

```diff
@@ -68,4 +69,5 @@ The list below shows the code systems used in the value sets above.
 |[SNOMED CT (Australian extension)](https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=snomed)|[Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1), [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1),[RANZCR Radiology Referral](ValueSet-ranzcr-radiology-referral.html)|NCTS|
 |[Tags for the Identification of Languages](https://terminology.hl7.org/CodeSystem-v3-ietf3066.html)|[Common Languages in Australia](https://healthterminologies.gov.au/fhir/ValueSet/common-languages-australia-2)|FHIR|
 |[TaskStatus](https://hl7.org/fhir/R4/codesystem-task-status.html)|[AU eRequesting Task Status](ValueSet-au-erequesting-task-status.html)|FHIR|
-{:.grid}
\ No newline at end of file
+|[v3.ActCoverageTypeCode](http://terminology.hl7.org/ValueSet/v3-ActCoverageTypeCode)|[AU eRequesting Coverage Type Self-Pay](ValueSet-au-erequesting-coverage-self-pay.html)|FHIR|
```

Comment:
Update to match the title: V3 Value SetActCoverageTypeCode

Reply:
This has not been addressed.

Reply:
The code system URL has been fixed but the hyperlink text should change to match the code system title "v3 Code System ActCode". Please change "v3 Code System ActCode" to "v3 Code System ActCode".

Reply:
Thanks - sorry about that - should be fixed now.

Reply:
Thanks, resolved now.